google/resource_compute_instance_test.go

```diff
@@ -1115,7 +1116,9 @@ func TestAccComputeInstance_secondaryAliasIpRange(t *testing.T) {
 func TestAccComputeInstance_hostname(t *testing.T) {
```

Comment:
slightly weird place, but was less disruptive to convert this test to use computeBeta than the basic* tests in this file

Reply:
👍